MentoAD.sln

```diff
@@ -9,6 +9,19 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HW03.Calculator.Static", "M
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HW03.Birthday", "MentoringHomeWork\HW3.0\HW03.Birthday\HW03.Birthday.csproj", "{6E851AD7-F6F1-4DBF-BEBB-D7922E194A3A}"
 EndProject
+<<<<<<< HEAD
```

Comment:
You tried to resolve merge conflicts here using UI probably - so these comments created an issue for you
![image](https://github.com/user-attachments/assets/b9acb902-5e3c-4c79-ba7a-652335027dc4)